### ISSUE TITLE
faulty "App wants additional access to your Google Account" prompt

### ISSUE THREAD

**Issue by vonovak** [google/GoogleSignIn-iOS#457]
**Describe the bug**
Upon repeated sign in, the SDK shows a prompt that reads "App wants additional access to your Google Account" even though no extra access is being requested. This is highly confusing for users.

Issue is reproducible in the sample app.

<details>
  <summary>screenshot</summary>

![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-16 at 22 45 02](https://github.com/user-attachments/assets/af3add0e-5d29-41c4-8164-38b308577d4b)

</details>

**To Reproduce**

- sign in
- sign out
- sign in again

See video

[youtube.com/shorts/O88Dw8pfdYU](https://youtube.com/shorts/O88Dw8pfdYU)

https://github.com/user-attachments/assets/3ef805e3-39de-473c-91b2-35afe30ef66b

**Expected behavior**

I don't expect to see the "App wants additional access to your Google Account" because I'm not asking for extra access.

**Screenshots**
see above

**Environment**
- Device: iphone 17.4 simulator

**Comment by gjbadros** [comment]
Yes, this is a big deal and a regression compared to months ago where it unnecessarily scares users into thinking something funny/risky is going on.  +1 for this being an important fix!

**Comment by mdmathias** [comment]
Thanks for the issue. We are investigating. I will update here as I learn more.

**Comment by vsk-pvr** [comment]
Same here

**Comment by georgeselkhoury** [comment]
same here

**Comment by vonovak** [comment]
It's clear at this point that there's an issue, and people are looking into it. Let's please not spam the issue and wait for actual updates. 
Thank you

**Comment by tehsven** [comment]
Hi @mdmathias thanks for looking into this. It's been over 2 weeks now - do you have any updates to share?

**Comment by RaymRaym** [comment]
Same here when using SwfitUI. Any updates?

**Comment by vonovak** [comment]
👋  thanks for response, does [this link](https://private-user-images.githubusercontent.com/1566403/349273006-3ef805e3-39de-473c-91b2-35afe30ef66b.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2Mjg5MjQsIm5iZiI6MTcyOTYyODYyNCwicGF0aCI6Ii8xNTY2NDAzLzM0OTI3MzAwNi0zZWY4MDVlMy0zOWRlLTQ3M2MtOTFiMi0zNWFmZTMwZWY2NmIubXA0P1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMjJUMjAyMzQ0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWU2MTdmMWE3MWQ3NTJiNWFlMzBlODkzOWMzOWQ1ZDUzMGZlZDdlZTA5MzZjYzQ0MTE1YmJhN2VlMDM5YmQ1MyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.KYhiY8-1PHOSps-8UE_WYsMJHhh-iJ8_r-8QDu__jaM) work?

**Comment by mdmathias** [comment]
Sadly no: "This private-user-images.githubusercontent.com page can’t be found"

**Comment by vonovak** [comment]
hello, I put it on Y: https://youtube.com/shorts/O88Dw8pfdYU

**Comment by vonovak** [comment]
hello @mdmathias,
Just flagging that it's been a while since the issue was opened, and it would be nice to get an update on this :) 

Thank you!

**Comment by vonovak** [comment]
I'm no longer seeing this error, so I'm closing the issue.
